data/Music/Sonic the Hedgehog Series/Sonic CD/You Can Do Anything.meta

```diff
@@ -0,0 +1,4 @@
+Sonic CD (JP) - You Can Do Anything

```

Comment:
A couple of notes:
- The Malon voice is kind of buried in the mix, especially near the end. You might want to bring this out more.
- Having the entire song come to an end and then start again feels strange. I would recommend finding another loop point so the music is not constantly interrupted.